pkg/database/migrations/v_0009/migrate.go

```diff
@@ -12,23 +16,81 @@ const Version = "2c2299e4e061"
 func Migrate(db *gorm.DB) error {
 	return migrations.RunWithoutForeignKeyIfNeeded(db, func() error {
 		return db.Transaction(func(tx *gorm.DB) error {
-			if err := tx.Migrator().AutoMigrate(&Context{}); err != nil {
-				return err
+			// Rename the existing metricx tables and drop indexes
+			tables := []string{"metrics", "latest_metrics"}
+			for _, table := range tables {
+				index := fmt.Sprintf("idx_%s_run_id", table)
+				if err := dropIndex(tx, table, index); err != nil {
+					return eris.Wrapf(err, "error dropping %s", index)
+				}
+				if err := tx.Migrator().RenameTable(table, backupName(table)); err != nil {
+					return eris.Wrapf(err, "error renaming %s", table)
+				}
+			}
+			index := "idx_metrics_iter"
+			if err := dropIndex(tx, backupName("metrics"), index); err != nil {
+				return eris.Wrapf(err, "error dropping %s", index)
 			}
```

Comment:
maybe we can make `tables` as a `map` like `map["metrics"] = [metrics_indexs]` and iterate over this map instead? in that everything will be a bit cleaner and more understandable and you don't need another `if` statement
```
if err := dropIndex(tx, backupName("metrics"), index); err != nil {
 				return eris.Wrapf(err, "error dropping %s", index)
 			}
```

Reply:
this case is weird because `idx_metrics_iter` only exists for `metrics`, not `latest_metrics` -- so it can't really be in the loop. but it could `map[string][]string` for table -> indexes I suppose

Reply:
yep yep, I talked about map of slices so you can specify own index list for each table and it will look a bit more clean.

Reply:
done